### DISCUSSION TITLE
Two questions about the settings with CDN, thx!

### DISCUSSION THREAD

**Discussion by shuaixf** [nuxt/nuxt#19102] [General]
1) Set the publicPath to a CDN url in the nuxt.config.js file.
2) nuxt build (not a static website, can't use nuxt generate). 

Questions:
1) .nuxt/dist directory doesn't contain the css/images/js directories and files from Static directory
2) the urls of the css/images/js didn't change to be the cdn links.

![image](https://user-images.githubusercontent.com/2179211/219546810-63a46483-b1eb-4a3d-a513-da2d440a9594.png)
![image](https://user-images.githubusercontent.com/2179211/219547305-1a22dcc2-9020-4437-b425-3d8022f9de9c.png)

How does the Static directory work with CDN, thx!!!

**Comment by mikespinks0401** [comment]
did you try setting app.cdnURL in the nuxt.config file? 
You can also set it as an environment variable with NUXT_APP_CDN_URL
